website/content/ja/wgs/platforms/_index.md

```diff
@@ -9,19 +9,21 @@ To participate join us on Slack at
 [#wg-platforms](https://cloud-native.slack.com/archives/C020RHD43BP)
 or the meetings described below.
 
+To register for updates and to "Join" the group, please also consider joining the [TAG App Delivery CNCF Community Group](https://community.cncf.io/tag-app-delivery/)
+
 ## Chairs
 
-* Josh Gavant (@joshgav)
-* Roberth Strand (@roberthstrand)
+* Technical Lead - Roberth Strand (@roberthstrand)
 * Abby Bangser (@abangser)
 * Colin Griffin (@krumware)
+* Atulpriya Sharma (@techmaharaj)
 
 ## Meetings
 
 * Meeting schedule: 2nd and 4th Tuesday of each month at [1600 Europe/London time](https://www.timeanddate.com/worldclock/converter.html?iso=20240514T150000&p1=136)
-    * [2nd Tuesday event](https://www.google.com/calendar/event?eid=cWE0ZHMyMThicDF1dDZsa3N2ZGlqYmt2Z2pfMjAyNDA1MTRUMTUwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&ctz=UTC)
-    * [4th Tuesday event](https://www.google.com/calendar/event?eid=NGhyOHY1ZWVrbDliODY3bXU5ZnRtYWo0ZGdfMjAyNDA1MjhUMTYwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&ctz=UTC)
-    * [Full CNCF calendar](https://calendar.google.com/calendar/u/0/embed?src=linuxfoundation.org_o5avjlvt2cae9bq7a95emc4740@group.calendar.google.com)
+* Please RSVP for meetings on the [TAG App Delivery Community Groups Page](https://community.cncf.io/tag-app-delivery/). This will also provide a calendar entry.
+* Meeting Location, Zoom: https://zoom-lfx.platform.linuxfoundation.org/meeting/94326372364?password=0681764d-b152-4295-8bd2-6b9fcf77bff1
 * Zoom: https://zoom-lfx.platform.linuxfoundation.org/meeting/94326372364?password=0681764d-b152-4295-8bd2-6b9fcf77bff1
     * Passcode: 77777
 * Agendas and notes: <https://docs.google.com/document/d/1_smeS9-j-SuHJi0VXjx4g9xiD2-tgqhnlwf5oSMDQgg>
+* Previous meetings can be viewed on the [TAG YouTube Channel](https://www.youtube.com/watch?v=eZYSQnsWRco&list=PLjNzvzqUSpxKH8X7wNfYZtkH_ARSeeQH0)
```

Comment:
Very nice to add this! Just missing on a couple of languages I think 🤔

Reply:
it's on the other languages, I just think it wasn't there on all of them, so the other languages might not show the diff